.github/workflows/gradle.yml

```diff
@@ -0,0 +1,32 @@
+# CI workflow
+
+name: Build
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
You can use list of events : 
        # Triggers the workflow on push or pull request events
        on: [push, pull_request]

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows